TelegramBot.py

```diff
@@ -355,7 +355,7 @@ def _cmd_turn_on_light(self, target_id):
                     'description': f"Turn on all lights", 
                     'url':f"/clb_turnon_light?entity_id=all"})
         for entity in statedict:
-            if re.match('^light.*', entity, re.IGNORECASE):
+            if re.match('^switch.on_off_plug_in_unit|^light.*', entity, re.IGNORECASE):
```

Comment:
Lets use wildcard at the end
```suggestion
            if re.match('^switch.on_off_plug_in_unit*|^light.*', entity, re.IGNORECASE):
```